src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -95,6 +106,29 @@ public void run() {
         });
     }
 
+    private String getGitReviewBranchName() {
+        String branchName = null;
+
+        String gitReviewFilePath = StringUtils.join(
+            new String[]{ProjectManager.getInstance().getOpenProjects()[0].getBasePath(), GITREVIEW_FILENAME}, File.separator);
+
+        File gitReviewFile = new File(gitReviewFilePath);
+        if (gitReviewFile.exists() && gitReviewFile.isFile()) {
+            try {
+                FileInputStream fileInputStream = new FileInputStream(gitReviewFilePath);
+
+                Properties properties = new Properties();
+                properties.load(fileInputStream);
+                branchName = properties.getProperty("defaultbranch");
+
+                fileInputStream.close();
+            } catch (IOException e) {
+            }
```

Comment:
If we really want to ignore this exception (which is an option in this case), please add a comment, or in this case if we return a Optional, just "Optional.absent()".